cumulus_library/cli.py

```diff
@@ -113,73 +113,126 @@ def export_all(self, study_dict: Dict):
             self.export_study(study_dict[key])
 
 
-def get_study_dict() -> Dict[str, PosixPath]:
+def get_abs_posix_path(path: str) -> PosixPath:
+    """Conveince method for hanlding abs vs rel paths"""
+    if path[0] == "/":
+        return Path(path)
+    else:
+        return Path(Path.cwd(), path)
+
+
+def create_template(path: str) -> None:
+    """Creates a manifest in target dir if one doesn't exist"""
+    abs_path = get_abs_posix_path(path)
+    manifest_path = Path(abs_path, "manifest.toml")
+    if manifest_path.exists():
+        sys.exit(f"A manifest.toml already exists at {abs_path}, skipping creation")
+    template_path = Path(
+        Path(__file__).resolve().parents[0], "studies/template/manifest.toml"
+    )
+    abs_path.mkdir(parents=True, exist_ok=True)
+    manifest_path.write_bytes(template_path.read_bytes())
+
+
+def get_study_dict(alt_paths: List) -> Optional[Dict[str, PosixPath]]:
     """Convenience function for getting directories in ./studies/
 
     :returns: A list of pathlib.PosixPath objects
     """
     manifest_studies = {}
-    library_path = Path(__file__).resolve().parents[0]
-    for path in Path(f"{library_path}/studies").iterdir():
-        if path.is_dir():
-            manifest_studies[path.name] = path
+    paths = [Path(Path(__file__).resolve().parents[0], "studies")]
+    if alt_paths is not None:
+        paths = paths + alt_paths
+    for parent_path in paths:
+        for child_path in parent_path.iterdir():
+            if child_path.is_dir():
+                manifest_studies[child_path.name] = child_path
+            elif child_path.name == "manifest.toml":
+                manifest_studies[parent_path.name] = parent_path
     return manifest_studies
 
 
 def run_cli(args: Dict):  # pylint: disable=too-many-branches
     """Controls which library tasks are run based on CLI arguments"""
-    builder = StudyBuilder(
-        args["s3_bucket"],
-        args["region"],
-        args["workgroup"],
-        args["profile"],
-        args["database"],
-    )
-    if args["verbose"]:
-        builder.verbose = True
+    if not args["build"] and not args["export"] and not args["create"]:
+        sys.exit(
+            (
+                "Expecting at least one of build, export or create as arguments. "
+                "See `cumulus_library --help` for more information"
+            )
+        )
+
+    if args["create"]:
+        create_template(args["path"])
```

Comment:
nit: If you just returned after this, you wouldn't need to modify the whole rest of this method.

nit: Honestly, this func would probably be more readable if you just did:
```
if create:
  do_create()
elif build:
  do_build()
elif export:
  do_export()
else:
  error out
```

And you could add a shared method like `make_builder(args)` to share that bit.

Reply:
makes sense for create, though it hurts my brain.

for build/export, i want to allow a user to do both of those in one go - i think that's a pretty common use case?

Reply:
Whatever works for you here, I was just throwing out suggestions.

It bumped my code senses that adding an extra mode here would cause a giant indentation change -- often a sign that code isn't maybe organized in the same way you are trying to use it. (not always true, just saying)

Reply:
my inclination for the time being is to leave it for now and if this gets more complex/that linting warning comes back, move things off into submethods then.

Reply:
oth the other thing i was trying to dodge here - i don't want to create a builder if it's not needed, just so i don't throw out an athena error trying to create a cursor if a user isn't to that point yet?